### DISCUSSION TITLE
IR tasmota just RAW functionality

### DISCUSSION THREAD

**Discussion by mumin50** [arendst/Tasmota#15259] [Q&A] [answered]
Hi All...
Is there a posibility to just enable IR Raw Sens and IR Raw recive without any decoding? if yes than please small howto.
I'm working with specific protocol and don't want to use uC time to decode other protocols 
If no where to find...

// You can also disable specific protocols in user_config_override
  // Example: disable some HVAC protocols to reduce flash size
  // #define DECODE_WHIRLPOOL_AC  false
  // #define SEND_WHIRLPOOL_AC    false
  // #define DECODE_SAMSUNG_AC    false
  // #define SEND_SAMSUNG_AC      false
  // ...
  
 ... whole HVAC protocol list to disable

**Comment by sfromis** [comment, marked as answer]
Well, you can find a lot by looking for `#ifdef` / `#if defined` in the source code. Example relevant file:
https://github.com/arendst/Tasmota/blob/development/lib/lib_basic/IRremoteESP8266/IRremoteESP8266/src/IRrecv.cpp
